inc/Engine/Optimization/RUCSS/Frontend/Subscriber.php

```diff
@@ -128,7 +130,8 @@ public function treeshake( string $html ) : string {
 		}
 
 		$html = $this->used_css->remove_used_css_from_html( $html, $used_css->unprocessedcss );
-		$html = $this->used_css->add_used_css_to_html( $html, $used_css->css );
+
+		$html = $this->used_css->add_used_css_to_html( $html, $used_css->css, $used_css->url, (bool) $used_css->is_mobile );
```

Comment:
Here are 4 params sent? Why not creating an array and pass a single array?

Reply:
I thought of sending the full DB row with all of the used_css details and use what is needed from it, what do u think?

Reply:
I think it's a good idea to send the row object. I don't like using array as parameter too much, because you need to add additional isset validation when using that.